### PR TITLE
feat: add lounge tier color tokens #110

### PR DESCRIPTION
**Pull request by ipin-photon**
# Alaska Airlines Pull Request

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

**Resolves:** #110

## Summary:

- Add color tokens for lounge and lounge plus
- The output will be (following the pattern from `one world` and `alaska elite`): 
```
$ds-color-tier-alaska-lounge: #01426a;
$ds-color-tier-alaska-loungeplus: #53b390;
```

## Type of change:

Please delete options that are not relevant.

- [x] New capability
- [ ] Revision of an existing capability
- [ ] Infrastructure change (automation, etc.)
- [ ] Other (please elaborate)

## Checklist:

- [x] My update follows the CONTRIBUTING guidelines of this project
- [x] I have performed a self-review of my own update

**By submitting this Pull Request, I confirm that my contribution is made under the terms of the Apache 2.0 license.**

_Pull Requests will be evaluated by their quality of update and whether it is consistent with the goals and values of this project. Any submission is to be considered a conversation between the submitter and the maintainers of this project and may require changes to your submission._

**Thank you for your submission!**<br>
-- Auro Design System Team

### CODE REVIEW

File: src/color/tier.json [hunk 1]
```diff
@@ -81,6 +81,22 @@
               "wcag": "n/a",
               "deprecated": false
             }
+          },
+          "lounge": {
+            "value": "{color.brand.midnight.400.value}",
+            "public": true,
+            "onDark": true,
```
Comment (blackfalcon) [inline src/color/tier.json:88]:
```suggestion
```

The `onDark` attribute is deprecated and SHOULD NOT be used on any new tokens. If the color is to be used on darker backgrounds, the new concept is `inverse`. 

I am assuming that this was copied for a previous token and left in.

File: src/color/tier.json [hunk 2]
```diff
@@ -81,6 +81,22 @@
...
+            "usage": "Longe tier color for Alaska Airlines",
+            "wcag": "n/a",
+            "deprecated": false
+          },
+          "loungeplus": {
+            "value": "{color.brand.loungeplus.value}",
+            "public": true,
+            "onDark": true,
```
Comment (blackfalcon) [inline src/color/tier.json:96]:
```suggestion
```

**Review by blackfalcon — CHANGES_REQUESTED** [review]

File: src/color/tier.json (hunk 1 above)
Comment (ipin-photon) [reply src/color/tier.json]: I've resolved your feedback. Please revalidate the changes. Thank you

File: src/color/tier.json (hunk 2 above)
Comment (ipin-photon) [reply src/color/tier.json]: I've resolved your feedback. Please revalidate the changes. Thank you

**Comment by blackfalcon** [comment]
Validated that the following tokens have been added to the repo: 

```css
--ds-color-brand-loungeplus: #53b390;

--ds-color-tier-alaska-lounge: #01426a;
--ds-color-tier-alaska-loungeplus: #53b390; //(alias for --ds-color-brand-loungeplus)
```

**Review by blackfalcon — CHANGES_REQUESTED** [review]
Please see CLA required step. 

https://github.com/AlaskaAirlines/AuroDesignTokens/pull/157#issuecomment-2106851936

**Comment by ipin-photon** [comment]
> Please see CLA required step.
> 
> [#157 (comment)](https://github.com/AlaskaAirlines/AuroDesignTokens/pull/157#issuecomment-2106851936)

I have updated commit author and already signed the CLA.

File: src/color/tier.json [hunk 3]
```diff
@@ -73,6 +73,20 @@
               "wcag": "n/a",
               "deprecated": false
             }
+          },
+          "lounge": {
+            "value": "{color.brand.midnight.400.value}",
```
Comment (jason-capsule42) [inline src/color/tier.json:78]: midnight is a deprecated tier 1 token name. Please work with @leeejune to make sure we are pointing this to the correct "new" tier 1 token name(s).

**Review by jason-capsule42 — CHANGES_REQUESTED** [review]

File: src/color/tier.json (hunk 3 above)
Comment (ipin-alaska) [reply src/color/tier.json]: Hi @jason-capsule42, to overcome that deprecated token name I added another color brand token named `lounge` with the value of `01426a`. Please let me know if there’s another concern.
CC. @leeejune

**Comment by jason-capsule42** [comment]
@ipin-alaska this looks good but it just needs the the branch updated with Master before it can be merged.

**Review by jason-capsule42 — APPROVED** [review]

**Comment by ipin-alaska** [comment]
> @ipin-alaska this looks good but it just needs the the branch updated with Master before it can be merged.

Okay @jason-capsule42, I just updated the branch with master.

**Review by jason-capsule42 — APPROVED** [review]

**Pull request merged**